utils/github_reporting.py

```diff
@@ -220,6 +225,46 @@ def get_pull_requests(self):
 
         return pr_list
 
+    def get_template_issues(self):
+        json_data = requests.get(self.get_job_api_url(details=True)).json()
+
+        test_templates = json_data['job']['settings'].get('TEST_TEMPLATES')
+        if not test_templates:
+            return []
+
+        test_templates = test_templates.split(" ")
+
+        all_templates = []
+
+        for log in json_data['job']['ulogs']:
+            if log == 'update-template-versions.txt':
+                log_file = "{}/tests/{}/file/{}".format(
+                    OPENQA_URL, self.job_id, log)
+
+                template_list = requests.get(log_file).text.split('\n')
+                for line in template_list:
+                    all_templates.append(
+                        re.sub(r"(.*)-([^-]*-[^-]*)\.noarch", r"\1 \2", line))
+
+        templates = []
+
+        for template_name in test_templates:
+            for package_name in all_templates:
+                if template_name in package_name:
```

Comment:
This won't work as one template name can be a prefix of another (like `fedora-30` and `fedora-30-minimal`). Better compare including end separator (replaced above with a space).